mk/VRBRAIN/config_vrubrain-v52_APM.mk

```diff
@@ -1,172 +1,10 @@
-#
-# Makefile for the VR MICRO BRAIN 5.2 APM configuration
-#
-
-#
-# Use the configuration's ROMFS.
-#
-ROMFS_ROOT	 = $(SKETCHBOOK)/mk/VRBRAIN/ROMFS_VRUBRAIN52_APM
-
-MODULES		+= $(APM_MODULE_DIR)
-
-#
-# Board support modules
-#
-MODULES		+= drivers/device
-MODULES		+= drivers/stm32
-MODULES		+= drivers/stm32/adc
-MODULES		+= drivers/stm32/tone_alarm
-MODULES		+= drivers/led
-MODULES		+= drivers/buzzer
-
-
-MODULES		+= drivers/boards/vrubrain-v52
-MODULES		+= drivers/vrbrain/vroutput
-MODULES		+= drivers/vrbrain/vrinput/controls
-MODULES		+= drivers/vrbrain/vrinput
-
-
-
-
-MODULES		+= drivers/mpu6000
-MODULES		+= drivers/hmc5883
-MODULES		+= drivers/hmc5983
-
-MODULES		+= drivers/ms5611
-MODULES		+= drivers/mb12xx
-
-
-
-
-
-
-
-MODULES		+= drivers/airspeed
-
-MODULES		+= drivers/meas_airspeed
-
-
-
-
-
-#
-# System commands
-#
-MODULES		+= systemcmds/mtd
-MODULES		+= systemcmds/bl_update
-
-
-
-
-
-MODULES		+= systemcmds/pwm
-MODULES		+= systemcmds/esc_calib
-MODULES		+= systemcmds/reboot
-MODULES		+= systemcmds/top
-
-
-MODULES		+= systemcmds/tests
-
-MODULES		+= systemcmds/nshterm
-
-#
-# General system control
-#
-
-
-
-
-
-
-#
-# Estimation modules (EKF/ SO3 / other filters)
-#
-
-
-
-
-
-
-#
-# Vehicle Control
-#
-
-
-
-
-
-
-
-
-#
-# Logging
-#
-
-
-#
-# Unit tests
-#
-
-
-
-#
-# Library modules
-#
-MODULES		+= modules/systemlib
-
-
-MODULES		+= modules/uORB
-
-#
-# Libraries
-#
-
-
-MODULES		+= lib/mathlib/math/filter
-
-
-
-MODULES		+= lib/conversion
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-#
-# Transitional support - add commands from the NuttX export archive.
-#
-# In general, these should move to modules over time.
-#
-# Each entry here is <command>.<priority>.<stacksize>.<entrypoint> but we use a helper macro
-# to make the table a bit more readable.
-#
-define _B
-	$(strip $1).$(or $(strip $2),SCHED_PRIORITY_DEFAULT).$(or $(strip $3),CONFIG_PTHREAD_STACK_DEFAULT).$(strip $4)
-endef
-
-#                  command                 priority                   stack  entrypoint
-BUILTIN_COMMANDS := \
-	$(call _B,  sercon,                 ,                          2048,   sercon_main                ) \
-	$(call _B,  serdis,                 ,                          2048,   serdis_main                ) \
-	$(call _B,  msconn,                 ,                          2048,   msconn_main                ) \
-	$(call _B,   msdis,                 ,                          2048,    msdis_main                ) \
-	$(call _B, sysinfo,                 ,                          2048,  sysinfo_main                )
+#
+# Makefile for the VRUBRAIN 5.2 APM configuration
+#
+include $(SKETCHBOOK)/mk/VRBRAIN/vrbrain_common.mk
+
+#MODULES		+= drivers/mpu9250
+MODULES		+= drivers/boards/vrubrain-v52
+MODULES		+= drivers/pwm_input
+#MODULES         += modules/uavcan
+MODULES         += lib/mathlib

```

Comment:
drivers/pwm_input and lib/mathlib seem to be common to all of them so they could go to vrbrain_common.mk too.